recipes-domu/domu-image-weston/domu-image-weston.bbappend

```diff
@@ -11,9 +11,9 @@ XT_QUIRK_BB_ADD_LAYER += "meta-xt-prod-extra"
 ################################################################################
 # Renesas R-Car
 ################################################################################
-SRC_URI_rcar = "repo://github.com/xen-troops/manifests;protocol=https;branch=vgpu-dev;scmdata=keep"
+SRC_URI_rcar = "repo://github.com/xen-troops/manifests;protocol=https;branch=master;manifest=prod_devel/domu.xml;scmdata=keep"
 XT_QUIRK_PATCH_SRC_URI_rcar = "file://${S}/meta-renesas/meta-rcar-gen3/docs/sample/patch/patch-for-linaro-gcc/0001-rcar-gen3-add-readme-for-building-with-Linaro-Gcc.patch;patchdir=meta-renesas"
-XT_BB_LAYERS_FILE_rcar = "meta-rcar-gen3/doc/bblayers.conf"
+XT_BB_LAYERS_FILE_rcar = "meta-xt-prod-extra/conf/bblayers.conf"
```

Comment:
> > bitbake only cares about build/conf/bblayers.conf

> That is the fact.

Well, I've looked through the bitbake manual and not really sure. 
From these parts [1](https://www.yoctoproject.org/docs/1.6/bitbake-user-manual/bitbake-user-manual.html#configuration-files) [2](https://www.yoctoproject.org/docs/1.6/bitbake-user-manual/bitbake-user-manual.html#parsing-the-base-configuration-metadata) it is not really clear if bitbake will totally ignore another .conf file in layer/conf folder.
So I would stay on the safe side and do not put the file in a path where bitbake browses by default.